BillTrack.Application/Services/SqsPublisher.cs

```diff
@@ -44,6 +48,34 @@ public async Task PublishMessageAsync<T>(string queueName, T message) where T :
             }
         };
 
-        await _sqs.SendMessageAsync(request);
+        try
+        {
+            await _sqs.SendMessageAsync(request);
+        }
+        catch (Exception ex)
+        {
+            throw;
```

Comment:
there is no point in making the `catch` without additional logic. `catch` + `throw` does the same as no `try` block was used.